query-builder/src/test/java/com/datastax/oss/driver/api/querybuilder/insert/RegularInsertTest.java

```diff
@@ -74,4 +74,10 @@ public void should_generate_if_not_exists_and_timestamp_clauses() {
     assertThat(insertInto("foo").value("a", bindMarker()).ifNotExists().usingTimestamp(1))
         .hasCql("INSERT INTO foo (a) VALUES (?) IF NOT EXISTS USING TIMESTAMP 1");
   }
+
+  @Test
+  public void should_generate_ttl_clause() {
+    assertThat(insertInto("foo").value("a", bindMarker()).usingTtl(10))
```

Comment:
I suggest that we add tests for when both `USING TIMESTAMP` and `USING TTL` are present.

Reply:
And also for `AND TTL`